a10_octavia/controller/worker/tasks/persist_tasks.py

```diff
@@ -0,0 +1,82 @@
+#    Copyright 2019, A10 Networks
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+
+import logging
+import acos_client.errors as acos_errors
+
+from a10_octavia.controller.worker.tasks.common import BaseVThunderTask
+from a10_octavia.controller.worker.tasks import utils
+from a10_octavia.common.a10constants import SP_OBJ_DICT, PERS_TYPE
+
+LOG = logging.getLogger(__name__)
+
+
+class HandleSessionPersistenceDelta(BaseVThunderTask):
+
+    def execute(self, vthunder, pool):
+        c_pers, s_pers, sp = None, None, None
```

Comment:
Remove this for 2 reason.
1. (Future ref): No need to define these as None like this if we are going to assign them right away. 
2. We don't use c_pers or s_pers

Reply:
I accidentally overwritten the old persist_task.py that was in my dev machine which was not cleaned up, and it went to PR, my bad!